bezier-rs/lib/src/lib.rs

```diff
@@ -688,6 +688,9 @@ impl Bezier {
 	/// 2. The on-curve point for `t = 0.5` must occur roughly in the center of the polygon defined by the curve's endpoint normals.
 	/// See [the offset section](https://pomax.github.io/bezierinfo/#offsetting) of Pomax's bezier curve primer for more details.
 	fn is_scalable(&self) -> bool {
+		if let BezierHandles::Linear = self.handles {
```

Comment:
Remove `let` and change `=` to `==`, there is no variable being bound here so this isn't a case for `if let`.